src/lib.rs

```diff
@@ -451,136 +488,165 @@ impl Config {
     /// falling back to the default configuration.
     pub fn get_compiler(&self) -> Tool {
         let opt_level = self.get_opt_level();
-        let debug = self.get_debug();
         let target = self.get_target();
-        let msvc = target.contains("msvc");
-        self.print(&format!("debug={} opt-level={}", debug, opt_level));
 
         let mut cmd = self.get_base_compiler();
+        // FIXME: this may detect nvcc if compiler happens to be in a directory containing nvcc but
+        // otherwise unrelated to nvcc, whatever it is.
```

Comment:
Yeah I'm not really sure what we can do here, I sorta forget what nvcc was even for as well (it was submitted awhile ago). I'd be fine to changing this to just check the basename, not the whole path.